wrappers/map.js

```diff
@@ -59,8 +59,8 @@ class Map extends EventEmitter {
         if (!config.popupFilter || config.popupFilter(feature)) {
 
           // don't bind the feature popup if it isn't defined in config
-          if (typeof config.popup(feature.properties) !== 'undefined') {
-            layer.bindPopup(config.popup(feature.properties))
+          if (typeof config.popupStyle(feature.properties) !== 'undefined') {
+            layer.bindPopup(config.popupStyle(feature.properties))
```

Comment:
If `popupStyle` isn't set and `notifies` is, will this attempt to call `popupStyle` anyway?

```js
if (config.popupStyle || config.notifies) {
  // ...
  if (typeof config.popupStyle(feature.properties) !== 'undefined') {
```